### PR TITLE
feat(GitHubActions): Add cypress yml config to run github actions

### PR DESCRIPTION
**Pull request by Fewwy**
Tested it on my fork and it works :)
image

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: cypress-test-run
+on: [push]
```
Comment (ikerreyes) [inline .github/workflows/cypress.yml:2]: To me this is good, but if we reach some limits or anything, we can consider alternatives.

**Review by ikerreyes — APPROVED** [review]
Great, thanks!

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (Fewwy) [reply .github/workflows/cypress.yml:2]: Yeah, we can check the https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows 
and choose what suits us better

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (gkarat) [inline .github/workflows/cypress.yml:9]: It seems that v3 is available (however, not sure what is the difference): https://github.com/actions/checkout

**Review by gkarat — APPROVED** [review]
looks good to me, let's merge and see how it works

**Pull request merged**